rdr_service/offline/genomic_pipeline.py

```diff
@@ -58,6 +58,15 @@ def c2_participant_workflow():
         controller.run_c2_participant_workflow()
 
 
+def c1_participant_workflow():
+    """
+    Entrypoint for Cohort 1 Participant Workflow,
+    Sources from Cohort 1 participants that have reconsented.
+    """
+    with GenomicJobController(GenomicJob.C2_PARTICIPANT_WORKFLOW) as controller:
```

Comment:
Should this be initialized with the C1 workflow id?

Reply:
Good catch, this was a copy/paste error. I've updated the code.